samples/dotnet/EdgeDownstreamDevice/EdgeDownstreamDevice.csproj

```diff
@@ -9,6 +9,6 @@
   </PropertyGroup>
 
   <ItemGroup>
-	<PackageReference Include="Microsoft.Azure.Devices.Client" Version="1.28.2" />
+	<PackageReference Include="Microsoft.Azure.Devices.Client" Version="1.41.2" />
```

Comment:
We use version 1.28.2 of the client SDK throughout the code in this branch. Are these samples (and Microsoft.Azure.WebJobs.Extensions.EdgeHub) the only projects triggering CG alerts for Newtonsoft?

Reply:
Also, maybe it would be better to update the transitive dependency (Newtonsoft.Json?) here, rather than upgrade the SDK.

Reply:
yes, for sure - we should not update the SDK version here.